.gitignore

```diff
@@ -1,5 +1,6 @@
 /vendor/
 /test/fixture/recipe/tmp/
-.php_cs.cache
 *.phar
 phpunit.xml
+coverage.xml
+.DS_Store
```

Comment:
This is os specific ignore. It should go into your system global ignore.

Reply:
OK. I'll revert this change. Thank you.